File: plik.py
# Funkcja do obliczania sumy dwóch liczb

def suma(a, b):
    
    return a + b  # Zwracamy wynik dodawania dwóch liczb


# Pobieramy dane wejściowe od użytkownika

num1 = float(input("Podaj pierwszą liczbę: "))  # Wczytujemy pierwszą liczbę
num2 = float(input("Podaj drugą liczbę: "))  # Wczytujemy drugą liczbę

# Obliczamy sumę

result = suma(num1, num2)  # Przechowujemy wynik dodawania w zmiennej result

# Wyświetlamy wynik

print(f"Suma {num1} i {num2} wynosi: {result}")  # Wyświetlamy wynik sumowania

def odejmowanie(a, b):
    
    # Zwracamy wynik odejmowania dwóch liczb

    return a - b  

# Pobieramy dane wejściowe od użytkownika

Num1 = float(input("Podaj pierwszą liczbę: "))  # Wczytujemy pierwszą liczbę
Num2 = float(input("Podaj drugą liczbę: "))  # Wczytujemy drugą liczbę

# Obliczamy sumę

result2 = suma(Num1, Num2)  # Przechowujemy wynik odejmowania w zmiennej result

# Wyświetlamy wynik

print(f"Suma {Num1} i {Num2} wynosi: {result2}")  # Wyświetlamy wynik sumowania

File: plik.php
<?php

// Ustawienie nagłówka strony na kodowanie UTF-8

header('Content-Type: text/html; charset=UTF-8');


// Funkcja do obliczenia sumy dwóch liczb

function sum($a, $b) {
   
    return $a + $b; // Zwracamy sumę dwóch liczb

}


// Sprawdzamy, czy formularz został wysłany

if ($_SERVER['REQUEST_METHOD'] === 'POST') {
  
    // Pobieramy dane z formularza
  
    $num1 = $_POST['number1'];
    $num2 = $_POST['number2'];
    
    // Wyświetlamy wynik sumy
   
    echo "Suma wynosi: " . sum($num1, $num2);
}

?>

<!-- Formularz do wpisania dwóch liczb -->

<form method="POST">
    
    <label for="number1">Liczba 1:</label>
    <input type="number" id="number1" name="number1"><br><br>
    
    <label for="number2">Liczba 2:</label>
    <input type="number" id="number2" name="number2"><br><br>
    
    <button type="submit">Oblicz sume</button>

</form>
